pkg/ccl/changefeedccl/changefeed_dist.go

```diff
@@ -308,7 +308,9 @@ func startDistChangefeed(
 			finishedSetupFn = func(flowinfra.Flow) { resultsCh <- tree.Datums(nil) }
 		}
 
-		jobsprofiler.StorePlanDiagram(ctx, execCfg.DistSQLSrv.Stopper, p, execCfg.InternalDB, jobID)
+		if log.V(1) {
```

Comment:
Hmm I understand that this could write too much data to the `system.job_info` table but this information is very useful to the CDC team when we're debugging support issues. It helps us visualize which nodes are participating in the changefeed and also is a proxy for when the changefeed replanned. My understanding is that in newer CRDB versions, the DistSQL diagram will truncate the number of ranges for each node shown on the diagram, which should help a lot with size?

Reply:
We discussed this offline and decided that logging the plan diagram URLs would be an acceptable alternative. This PR adds the logging code: https://github.com/cockroachdb/cockroach/pull/126289.

Reply:
We discussed this offline and decided that logging the plan diagram URLs would be an acceptable alternative. This PR adds the logging code: #126289.